integration-tests/environments/node/index.cjs

```diff
@@ -18,25 +18,14 @@
 
 const os = require("node:os");
 const { Client } = require("mocha-remote-client");
-const fs = require("fs-extra");
-const path = require("path");
-const v8 = require("v8");
-const vm = require("vm");
-
-v8.setFlagsFromString("--expose_gc");
 
 const client = new Client({
   title: `Node.js v${process.versions.node} on ${os.platform()} (via CommonJS)`,
   tests(context) {
-    // Exposing the Realm constructor as a global
-    global.fs = fs;
-    global.path = path;
-    global.environment = { ...context, node: true };
-    global.gc = vm.runInNewContext("gc");
```

Comment:
The main idea behind this refactor is to move this into a shared "node/src/setup-globals.ts" used across these two "runners" and the test ran with bare mocha (without mocha-remote - used when debugging native code).